src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.DefaultProxyCredentials.cs

```diff
@@ -37,7 +37,7 @@ public void SetGet_Roundtrips()
         {
             using (HttpClientHandler handler = CreateHttpClientHandler())
             {
-                var creds = new NetworkCredential("username", "password", "domain");
+                NetworkCredential creds = TestSecretHelper.CreateTestCredential("username", "password", "domain");
```

Comment:
Same question for all of these.  I'm surprised passing const string arguments to parameters named "username" and "password" doesn't cause alarms.

Reply:
User name is never an issue. The tool focus on password in specific context. It this case the password is a string literal in the `NetworkCredential` ctor call.